packages/scene-composer/src/components/panels/scene-components/AnimationComponentEditor.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useCallback, useContext } from 'react';
+
+import { useStore, ISceneNodeInternal } from '../../../store';
+import { sceneComposerIdContext } from '../../../common/sceneComposerIdContext';
+
+import { AnimationViewStateEditor, AnimationViewStateEditorProps } from './AnimationViewStateEditor';
+
+export interface AnimationComponentEditorProps extends AnimationViewStateEditorProps {
+  node: ISceneNodeInternal;
+  component: any;
+}
+export const AnimationComponentEditor: React.FC<AnimationViewStateEditorProps> = ({
+  onUpdate: _onUpdate,
```

Comment:
`AnimationViewStateEditorProps` has an extra `onUpdate` prop than `AnimationComponentEditorProps`, so `AnimationComponentEditorProps` should be the base type and `AnimationViewStateEditorProps` extends from it and add the extra `onUpdate` to itself